src/content/data-streams/release-notes.mdx

```diff
@@ -12,8 +12,8 @@ whatsnext:
 
 import { Aside } from "@components"
 
-<Aside type="note" title="Early Access">
-  Data Streams is available on Arbitrum Mainnet and Arbitrum Goerli in Early Access.{" "}
+<Aside type="note" title="MAINNET EARLY ACCESS">
```

Comment:
```suggestion
<Aside type="note" title="Mainnet Early Access">
```

Make this consistent even though callout titles are converted to all caps in the public docs. If the style changes in the future, it's easier to maintain.